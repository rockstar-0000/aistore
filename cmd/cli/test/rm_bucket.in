ais bucket create ais://$BUCKET_1 | awk '{print $1 " " $2}'
ais bucket rm ais://$BUCKET_1

ais bucket rm hdfs://tmp // FAIL "bucket "hdfs://tmp" does not exist"

ais bucket rm aws://$BUCKET_2 // FAIL "not supported"
ais bucket rm gcp://$BUCKET_2 // FAIL "not supported"
