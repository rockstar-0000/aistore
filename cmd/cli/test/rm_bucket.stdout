"ais://$BUCKET_1" created
"ais://$BUCKET_1" destroyed
